Sources/Core/Extensions/Other/XCTestCase+PovioKit.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  XCTestCase+PovioKit.swift
+//  PovioKit
+//
+//  Created by Gentian Barileva on 09/09/2022.
+//  Copyright © 2022 Povio Inc. All rights reserved.
+//
+
+import XCTest
+
+public extension XCTestCase {
+  func XCTAssertEqualURLRequest(_ lhs: URLRequest?, _ rhs: URLRequest?, file: StaticString = #filePath, line: UInt = #line) {
```

Comment:
Do we want to have the `NotEqual` method as well? 🤔

Reply:
I'm not sure honestly, maybe we can add it later on if such a need appears.

What do you think?

Reply:
It is useful in some cases. And to be consistent with XCTest framework, maybe we should add it 🤷‍♂️ . I don't have strong feelings about it though.